pyiron_continuum/fenicsx/plot.py

```diff
@@ -27,7 +31,7 @@ class PlotDeformed:
     def __init__(self):
         super().__init__()
 
-    def plotDefomed_functionspae(V, uh, factor):
+    def plotDefomed_functionspace(V, uh, factor):
```

Comment:
I would find it nicer if you stick to the snake case (at least I wouldn't mix snake and camel case)

Reply:
Yeah, I'll have to change all of this at some point any way. I just came up with whatever names I wanted on the fly (not more than 0.5s of brain power invested into it XD). There are also lots of redundant arguments in the functions that are probably not needed. All of this will be polished when we start synchronizing stuff between fenicsx and other continuum codes.